parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -47,8 +46,6 @@ use config::{
 	SYNC_COMMITTEE_SIZE,
 };
 
-const_assert!(SYNC_COMMITTEE_BITS_SIZE == SYNC_COMMITTEE_SIZE / 8);
```

Comment:
Just curious is that already calculated and checked [here](https://github.com/Snowfork/snowbridge/blob/046fcc1ab7eacd4163dbe51b46204ddeccb33386/parachain/pallets/ethereum-beacon-client/src/config/mainnet.rs#L5)

Reply:
Yeah, its calculated correctly. I just to be extra cautious though, in case someone (like a new developer) changes the config without knowing.

This is defensive approach.

Reply:
Though can probably move the check to `config/mod.rs`.